### PR TITLE
Specify netlify build

### LINKED ISSUE #1135: Move toolkit.fluxcd.io to netlify
In preparation for moving the docs into `f/website`, @stefanprodan suggested we move the docs to Netlify, so we can fix up redirects between the sites within Netlify more easily.

### PR DESCRIPTION
**Pull request by dholbach**
Related to: #1135

Follow https://www.starfallprojects.co.uk/posts/deploy-mkdocs-netlify/ to eventually fix #1135.

I realise this litters the main directory somewhat, but I hope that once the publication fully works and we turn the site into redirects, we can remove these files again. So only a temporary measure.

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[build]
+  command = "mkdocs build"
+  publish = "site"
```
Comment (hiddeco) [inline netlify.toml:3]: If this is a target directory, can you please include it in the `.gitignore`?

**Comment by dholbach** [comment]
https://deploy-preview-1353--gracious-northcutt-ac416e.netlify.app/ looks good - we'll just need to 

- [ ] stop the deploy to gh-pages
- [ ] update GH settings accordingly
- [ ] change the dns settings afterwards

**Comment by dholbach** [comment]
@hiddeco @stefanprodan ^ does the above sound good to you?

**Comment by hiddeco** [comment]
Sounds OK to me.

I can switch the DNS settings for you post merge, we can then remove the old site automation (and branch) once most of the world has the updated DNS settings.

**Comment by dholbach** [comment]
Perfect - thanks Hidde!

**Review by hiddeco — APPROVED** [review]
Based on the preview (and the things I reviewed in Netlify) this all looks OK, or manageable if something little ends up wrong.

Thanks a bunch @dholbach :tea:

**Pull request merged**